tasks/kernel_matrix_testing/vmconfig.py

```diff
@@ -190,6 +194,86 @@ def get_image_list(distro: bool, custom: bool) -> list[list[str]]:
     return table
 
 
+def get_local_image_list(distro: bool, custom: bool) -> list[list[str]]:
+    headers = [
+        "VM name",
+        "OS ID",
+        "OS Name",
+        "OS Version",
+        "Kernel",
+        "Architecture",
+        "Alternative names",
+        "Last Update",
+        "Up to Date",
+    ]
+
+    custom_kernels: list[list[str]] = []
+    for k in sorted(kernels, key=lambda x: tuple(map(int, x.split('.')))):
+        if lte_414(k):
+            custom_kernels.append([f"custom-{k}", "debian", "Debian", "Custom", k, "x86_64", "", "N/A", "N/A"])
+        else:
+            custom_kernels.append([f"custom-{k}", "debian", "Debian", "Custom", k, "x86_64", "", "N/A", "N/A"])
+            custom_kernels.append([f"custom-{k}", "debian", "Debian", "Custom", k, "arm64", "", "N/A", "N/A"])
+
+    local_distro_kernels: list[list[str]] = []
+    platforms = get_platforms()
+
+    # Group kernels by name and kernel version, show whether one or two architectures are supported
+    for arch in KMT_SUPPORTED_ARCHS:
+        for name, platinfo in platforms[arch].items():
+            if isinstance(platinfo, str):
+                continue  # Old format
+
+            # Check if image is available locally
+            if "image" in platinfo and "image_version" in platinfo:
```

Comment:
nit: negate the condition and continue to avoid excessive nesting 

```suggestion
            if "image" not in platinfo or "image_version" not in platinfo:
                continue
```